packages/lexical/src/LexicalEditor.ts

```diff
@@ -1302,12 +1302,18 @@ export class LexicalEditor {
       // This ensures that iOS does not trigger caps lock upon focus
       rootElement.setAttribute('autocapitalize', 'off');
       updateEditorSync(this, () => {
-        const selection = $getSelection();
+        let selection = $getSelection();
```

Comment:
This isn't re-used anywhere with the suggestion below

```suggestion
        const selection = $getSelection();
```